internal/controller/resource_manager.go

```diff
@@ -120,15 +114,19 @@ func (r *ResourceManager) watchEvents(ctx context.Context, resource string, apiG
 			r.logger.Errorw("Cannot convert object metadata", "error", err)
 			return err
 		}
-		wasUniqueEvent, err := uniqueEvent(*convertedData, &cache, event)
-		if err != nil {
-			r.logger.Errorw("Failed to hash object metadata", "error", err)
-			return err
-		}
+		// wasUniqueEvent, err := uniqueEvent(*convertedData, &cache, event)
+		// if err != nil {
+		// 	r.logger.Errorw("Failed to hash object metadata", "error", err)
+		// 	return err
+		// }
+		// if watch.Deleted == event.Type {
+		// 	fmt.Println(wasUniqueEvent)
+		// 	fmt.Println(metadataObj)
+		// }
```

Comment:
is there a reason why these are commented our rather than just deleted?

Reply:
Should be deleted!